mypy_django_plugin/transformers/managers.py

```diff
@@ -244,7 +245,10 @@ def _replace_type_var(ret_type: MypyType, to_replace: str, replace_by: MypyType)
     """
     if isinstance(ret_type, TypeVarType) and ret_type.fullname == to_replace:
         return replace_by
-    elif isinstance((instance := get_proper_type(ret_type)), Instance):
+
+    ret_type = copy_type(get_proper_type(ret_type))
+
+    if isinstance((instance := ret_type), Instance):
```

Comment:
`ret_type` is now `ProperType` here, meaning we should be able to remove the `isinstance(ret_type, ProperType)` on the remaining if-statements.

Reply:
👍